draft-ietf-rats-architecture.md

```diff
@@ -117,39 +119,40 @@ Appraisal Policy for Attestation Result:
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the Evidence it has appraised
 
 Attester:
 
-: An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
+: An entity (typically a device), whose Evidence must be appraised in order to determine whether it is considered trustworthy, such as when deciding whether it is authorized to perform some operation
 
 Endorsement:
 
-: A secure statement that some entity (typically a manufacturer) vouches for the integrity of an Attester's signing capability
+: Statements that Endorsers make (typically a manufacturer) that vouches for the design and implementation of the Attester. Often this includes statements about the integrity of an Attester's signing capability
 
 Endorser:
 
-: An entity that creates Endorsements that can be used to help to appraise the trustworthiness of Attesters
+: An entity (typically a manufacturer) whose Endorsements help Verifiers appraise the authenticity of Evidence
 
 Evidence:
 
-: A set of information about an Attester that is to be appraised by a Verifier
+: A set of information that asserts the trustworthiness status of an Attester, that is appraised by a Verifier
 
 Relying Party:
 
-: An entity that depends on the validity of information about another entity, typically for purposes of authorization.  Compare /relying party/ in {{RFC4949}}
+: An entity (typically an application), that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}
 
 Relying Party Owner:
 
-: An entity, such as an administrator, that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party.
+: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party
 
 Verifier:
 
-: An entity that appraises the validity of Evidence about an Attester
+: An entity (typically a service provider), that appraises the validity of Evidence about an Attester
```

Comment:
I don't think we should introduce the term "service provider" here, we'd have to define it.

Reply:
Laurence suggests "service" instead of "service provider"